lib/sqlalchemy/ext/linter.py

```diff
@@ -0,0 +1,114 @@
+import collections
+import itertools
+
+from sqlalchemy import util
```

Comment:
**mike bayer** (zzzeek) wrote:

these should be relative imports

Reply:
Done